src/Components/Patient/DailyRoundListDetails.tsx

```diff
@@ -11,23 +11,29 @@ const Loading = lazy(() => import("../Common/Loading"));
 const symptomChoices = [...SYMPTOM_CHOICES];
 const currentHealthChoices = [...CURRENT_HEALTH_CHANGE];
 
+function useConsultationQuery(consultationId: any, id: any) {
+  return useQuery(routes.getDailyReport, {
+    pathParams: {
+      consultationId,
+      id,
+    },
+  });
+}
```

Comment:
Any reason why it's wrapped in `useConsultationQuery`? Couldn't this be inlined inside the component?

Also, the naming: `ConsultationQuery` does not make sense with what it does. It fetches daily rounds and not consultation details right?

Reply:
I thought it would be better if i encapsulate the useQuery inside a custom hook so that it can take any necessary parameters and all data fetching actions.